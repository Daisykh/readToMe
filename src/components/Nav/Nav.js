import React from 'react';
import './Nav.css';
import { NavLink } from 'react-router-dom';

export const Nav = () => {
  return (
    <div className="Nav">
      <nav>
        <div id="menuToggle">
          <input type="checkbox" />

          <span></span>
          <span></span>
          <span></span>

          <ul id="menu">
            <li><NavLink to='/'>Home</NavLink></li>
            <li><NavLink to='/bookshelf'>My Bookshelf</NavLink></li>
            <li><NavLink to='/bookshelf'>Favorites</NavLink></li>
          </ul>
        </div>
      </nav>
    </div>
  );
};


export default Nav;