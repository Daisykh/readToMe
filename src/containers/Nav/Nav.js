import React, { Component } from 'react'
import './Nav.css'
import { NavLink, withRouter, Link } from 'react-router-dom';

class Nav extends Component {
  render() {
    return (
      <div className="Nav">
       <nav role="navigation">
        <div id="menuToggle">
         <input type="checkbox" />
 
            <span></span>
            <span></span>
            <span></span>

            <ul id="menu">
              <li><NavLink to='/'>Splash</NavLink></li>
              <li><NavLink to='/first'>Test</NavLink></li>
              <li><NavLink to='/first'>Test</NavLink></li>
              <li><NavLink to='/first'>Test</NavLink></li>
              <li><NavLink to='/first'>Test</NavLink></li>
            </ul>
          </div>
        </nav>
      </div>
    );
  }
}

export default Nav;