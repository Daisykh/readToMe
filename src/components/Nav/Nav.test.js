import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav';

describe('Nav', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<Nav />);

    expect(renderedComponent).toMatchSnapshot();
  })
})
