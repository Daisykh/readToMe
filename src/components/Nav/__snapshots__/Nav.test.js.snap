// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav
        role="navigation"
>
        <div
                id="menuToggle"
        >
                <input
                        type="checkbox"
                />
                <span />
                <span />
                <span />
                <ul
                        id="menu"
                >
                        <li>
                                <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/"
                                >
                                        Home
                                </NavLink>
                        </li>
                        <li>
                                <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                                >
                                        My Bookshelf
                                </NavLink>
                        </li>
                        <li>
                                <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                                >
                                        Favorites
                                </NavLink>
                        </li>
                </ul>
        </div>
</nav>,
      "className": "Nav",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          id="menuToggle"
>
          <input
                    type="checkbox"
          />
          <span />
          <span />
          <span />
          <ul
                    id="menu"
          >
                    <li>
                              <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/"
                              >
                                        Home
                              </NavLink>
                    </li>
                    <li>
                              <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                              >
                                        My Bookshelf
                              </NavLink>
                    </li>
                    <li>
                              <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                              >
                                        Favorites
                              </NavLink>
                    </li>
          </ul>
</div>,
        "role": "navigation",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              type="checkbox"
/>,
            <span />,
            <span />,
            <span />,
            <ul
              id="menu"
>
              <li>
                            <NavLink
                                          activeClassName="active"
                                          ariaCurrent="true"
                                          to="/"
                            >
                                          Home
                            </NavLink>
              </li>
              <li>
                            <NavLink
                                          activeClassName="active"
                                          ariaCurrent="true"
                                          to="/bookshelf"
                            >
                                          My Bookshelf
                            </NavLink>
              </li>
              <li>
                            <NavLink
                                          activeClassName="active"
                                          ariaCurrent="true"
                                          to="/bookshelf"
                            >
                                          Favorites
                            </NavLink>
              </li>
</ul>,
          ],
          "id": "menuToggle",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "type": "checkbox",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/"
                  >
                                    Home
                  </NavLink>
</li>,
                <li>
                  <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/bookshelf"
                  >
                                    My Bookshelf
                  </NavLink>
</li>,
                <li>
                  <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/bookshelf"
                  >
                                    Favorites
                  </NavLink>
</li>,
              ],
              "id": "menu",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/"
>
                    Home
</NavLink>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "ariaCurrent": "true",
                    "children": "Home",
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Home",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/bookshelf"
>
                    My Bookshelf
</NavLink>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "ariaCurrent": "true",
                    "children": "My Bookshelf",
                    "to": "/bookshelf",
                  },
                  "ref": null,
                  "rendered": "My Bookshelf",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/bookshelf"
>
                    Favorites
</NavLink>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "ariaCurrent": "true",
                    "children": "Favorites",
                    "to": "/bookshelf",
                  },
                  "ref": null,
                  "rendered": "Favorites",
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav
          role="navigation"
>
          <div
                    id="menuToggle"
          >
                    <input
                              type="checkbox"
                    />
                    <span />
                    <span />
                    <span />
                    <ul
                              id="menu"
                    >
                              <li>
                                        <NavLink
                                                  activeClassName="active"
                                                  ariaCurrent="true"
                                                  to="/"
                                        >
                                                  Home
                                        </NavLink>
                              </li>
                              <li>
                                        <NavLink
                                                  activeClassName="active"
                                                  ariaCurrent="true"
                                                  to="/bookshelf"
                                        >
                                                  My Bookshelf
                                        </NavLink>
                              </li>
                              <li>
                                        <NavLink
                                                  activeClassName="active"
                                                  ariaCurrent="true"
                                                  to="/bookshelf"
                                        >
                                                  Favorites
                                        </NavLink>
                              </li>
                    </ul>
          </div>
</nav>,
        "className": "Nav",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            id="menuToggle"
>
            <input
                        type="checkbox"
            />
            <span />
            <span />
            <span />
            <ul
                        id="menu"
            >
                        <li>
                                    <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/"
                                    >
                                                Home
                                    </NavLink>
                        </li>
                        <li>
                                    <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/bookshelf"
                                    >
                                                My Bookshelf
                                    </NavLink>
                        </li>
                        <li>
                                    <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/bookshelf"
                                    >
                                                Favorites
                                    </NavLink>
                        </li>
            </ul>
</div>,
          "role": "navigation",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                type="checkbox"
/>,
              <span />,
              <span />,
              <span />,
              <ul
                id="menu"
>
                <li>
                                <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/"
                                >
                                                Home
                                </NavLink>
                </li>
                <li>
                                <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/bookshelf"
                                >
                                                My Bookshelf
                                </NavLink>
                </li>
                <li>
                                <NavLink
                                                activeClassName="active"
                                                ariaCurrent="true"
                                                to="/bookshelf"
                                >
                                                Favorites
                                </NavLink>
                </li>
</ul>,
            ],
            "id": "menuToggle",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "type": "checkbox",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/"
                    >
                                        Home
                    </NavLink>
</li>,
                  <li>
                    <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                    >
                                        My Bookshelf
                    </NavLink>
</li>,
                  <li>
                    <NavLink
                                        activeClassName="active"
                                        ariaCurrent="true"
                                        to="/bookshelf"
                    >
                                        Favorites
                    </NavLink>
</li>,
                ],
                "id": "menu",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <NavLink
                      activeClassName="active"
                      ariaCurrent="true"
                      to="/"
>
                      Home
</NavLink>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "activeClassName": "active",
                      "ariaCurrent": "true",
                      "children": "Home",
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": "Home",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <NavLink
                      activeClassName="active"
                      ariaCurrent="true"
                      to="/bookshelf"
>
                      My Bookshelf
</NavLink>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "activeClassName": "active",
                      "ariaCurrent": "true",
                      "children": "My Bookshelf",
                      "to": "/bookshelf",
                    },
                    "ref": null,
                    "rendered": "My Bookshelf",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <NavLink
                      activeClassName="active"
                      ariaCurrent="true"
                      to="/bookshelf"
>
                      Favorites
</NavLink>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "activeClassName": "active",
                      "ariaCurrent": "true",
                      "children": "Favorites",
                      "to": "/bookshelf",
                    },
                    "ref": null,
                    "rendered": "Favorites",
                    "type": [Function],
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "div",
        },
        "type": "nav",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
